drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -5,8 +5,23 @@ function DragAndDropTemplates(configuration) {
     var ngettext;
     if ('DragAndDropI18N' in window) {
         // Use DnDv2's local translations
-        gettext = window.DragAndDropI18N.gettext;
-        ngettext = window.DragAndDropI18N.ngettext;
+        gettext = function(string) {
+            let translated = window.DragAndDropI18N.gettext(string);
+            // if DnDv2's translation is the same as the input, check if global has a different value
+            // This is useful for overriding the XBlock's string by themes (only for English)
+            if (string === translated && 'gettext' in window) {
+                translated = window.gettext(string);
+            }
+            return translated;
+        }
+        ngettext = function(strA, strB, n) {
+            let translated = window.DragAndDropI18N.ngettext(strA, strB, n);
+            let string = n == 1 ? strA : strB;
```

Comment:
What about languages with pluralization rules different than English?  It seems like there's a chance that ngettext could return the other input in that case, and we'd miss the fact that the text wasn't translated because we checked for equality with the wrong input.

Reply:
@jmbowman Sorry for the delay here. I was off for a week and couldn't respond immediately.

For other languages the `translated` value would be different than the `string` value so `if (string === translated && 'gettext' in window) ` would fail and it would return the `translated` value, as we would be comparing some other language with one of the input English strings. So, I think it should work fine.

Reply:
FYI: @jmbowman will be out for a couple of weeks, so there could be a delay.

Reply:
Fair, the logic bug I was thinking of would only prevent use of the fallback when you want to use it, but if we only want it when the language is English then that's a moot point.